Formula/ncmpcpp.rb

```diff
@@ -31,6 +31,7 @@ class Ncmpcpp < Formula
   depends_on "pkg-config" => :build
   depends_on "libmpdclient"
   depends_on "readline"
+  depends_on "ncurses"
```

Comment:
Please inquire upstream whether dropping support for macOS's ncurses is intentional, as that's something we almost always consider a porting issue that indicates lack of full macOS support and a significant regression from the status quo.